microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,436 +1,163 @@
-
-Visual hosting allows for (and requires) more granular control of layout.  When using this approach, the app needs specific handling of window management and rendering APIs.
-
-With visual hosting, for example you must choose how when the user resizes the window, how the webview scales in relation to the whole page - for example, if you want the webview to scale twice as much as the app.
+The supported API lists are linked to in the sections below:
+* [APIs for Windowed hosting](#apis-for-windowed-hosting)
+* [APIs for Window to Visual hosting](#apis-for-window-to-visual-hosting)
+* [APIs for Visual hosting](#apis-for-visual-hosting)
 
 
 <!-- ====================================================================== -->
-## Compatibility and constraints
-
-Key compatibility limitations include the operating system and rendering in framework and non-framework apps.
-
+## Windowed hosting: For displaying content quickly and easily
 
-<!-- ------------------------------ -->
-#### Operating systems
+_Windowed hosting_ means that in your app, WebView2 content is hosted directly in a window; that is, an HWND. The WebView2 HWND inherits many default properties from the operating system (OS).  The WebView2 control takes input from the OS, and the OS sends the input to the WebView2 control.  You can have multiple HWNDs in your app that will each be used as a WebView component to access web content.
 
-Windows 7 and Windows 8 can only do windowed hosting, not visual hosting.
+The benefit of this is that some of the Input/Output commands are handled for you by the OS or by the framework. However, you will still need to handle some aspects of window management.
 
-See [Windows 7 and 8](../index.md#windows-7-and-8) in _Introduction to Microsoft Edge WebView2_.
+For general information about Window management and `HWND` functionality, see [About Windows](/windows/win32/winmsg/about-windows).
 
 
 <!-- ------------------------------ -->
-#### Rendering WebView2 in framework and non-framework apps
+#### Advantages
 
-If you're using a UI framework for your app, you should use the corresponding WebView2 element for that UI framework.  If you aren't using a UI framework for your app (such as Win32 or React Native) or your UI framework doesn't have a WebView2 element, you'll need to create `CoreWebView2Controller` and render it into the desired app.
+* Windowed hosting allows for a solution that quickly displays web content without having to include features for inputs, outputs, and accessibility.
 
-**Note:** If your app's UI is built using `DirectComposition` or `Windows.UI.Composition`, then you should use `CoreWebView2CompositionController`, otherwise you should use `CoreWebView2Controller`.
+* Zooming and rasterization scales (such as menus and context menus) automatically scale to the app's parent `HWND`.  Windowed hosting also handles how the WebView control manages being focused and tabbing in/out of itself when it reaches the final element.
```

Comment:
nit: can this be split into different list items?

Reply:
Nic's comment breaks this list item into two items and improves wording.  Closing present.